index.js

```diff
@@ -92,6 +96,26 @@ class Conf {
 		this.store = plainObject();
 	}
 
+	migrate(options, expectedVersion) {
+		if (options.migrations) {
+			const runningVersion = this.store.packageVersion || '0.0.0';
+
+			if (semver.lt(runningVersion, expectedVersion)) {
+				const migrationsToRun = Object.keys(options.migrations).filter(version => {
+					return semver.lte(version, expectedVersion) && semver.gt(version, runningVersion);
+				}).sort(semver);
+
+				for (const version of migrationsToRun) {
+					options.migrations[version](this);
+				}
+			}
+
+			if (runningVersion !== expectedVersion) {
+				this.set('packageVersion', expectedVersion);
```

Comment:
You can't just set a key. You could potentially silently be overwriting a user's key by the same name. The key must me something that is very unlikely to conflict. Maybe `__packageVersion__`